packages/pwa-kit-dev/src/utils/test-fixtures/minimal-built-app/ssr.js

```diff
@@ -0,0 +1,6 @@
+/*
+ * Copyright (c) 2021, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
```

Comment:
Is this the `hello world` you mentioned previously? So we're only wanting to check that an `ssr.js` exists at all and not verify it has contents or does anything?

Reply:
Nope – that hello world is still incoming. The tests for `npm push` need a little sample artifact.

That hello-world will need to be a new package, in all likelihood.